packages/cashier/src/Components/eusdt-side-note.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Modal } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+const EUSDTSideNote = () => {
+    const [is_modal_open, setModalOpen] = React.useState(false);
+
+    const handleLearnMore = () => {
+        setModalOpen(!is_modal_open);
+    };
+
+    return (
+        <div className='cashier__usdt-side-note'>
+            <h2 className='cashier__usdt-side-note--header'>
+                <Localize i18n_default_text='About Tether' />
+            </h2>
+            <p>
```

Comment:
Could we use the new `Text` component to replace all the `p` elements in the file and pass in props as needed for styling?